README.md

```diff
@@ -68,7 +68,35 @@ One can create a new header file by using the command: `javac -h . ED25519.java`
 
 The output will be a header file, the contents hereof must be matched appropriately as described in the [documentation](https://docs.rs/jni/0.19.0/jni/) into the [lib.rs](crypto-jni/src/lib.rs) rust source file.
 
+## Android
+To build the android AAR package, one has to do the following:
 
+1. Clone this repository and remember to update git submodules, for example when cloning via
+```bash
+git clone https://github.com/Concordium/concordium-java-sdk.git --recurse-submodules
+```
+2. Set the ANDROID_HOME environment variable to the path to your android SDK installation.
+3. Run `make add-android-targets` from the root of this repository.
+4. Run `make android` from the root of this repository.
+5. Run `mvn install` from the root of the [concordium-android-sdk](./concordium-android-sdk) folder.
+
+`make add-android-targets` adds the rust targets that the native libraries will be built for.
+`make android` builds the native libraries for various targets.
+And the final step builds the aar file `concordium-android-sdk.aar` in the [target](./concordium-android-sdk/target)  folder, which can be used in android projects.
+
+For the library to work on android, the following packages must be added to the project (Shown in the gradle style)
```

Comment:
Would it make sense to build a package where the dependencies are included?

Reply:
No, As far as I can tell, it seems against best practice, and we should look to deploy it on maven and declare the dependencies there. 
But I just added like this for now. :thinking: